vignettes/goodness-of-fit.Rmd

```diff
@@ -139,9 +143,11 @@ rmarkdown::render(workflow1$reportFileName)
 The output report for Example 1 is shown below. 
 There are only 2 figures in the report (linear and log scales) showing the time profile of the output path '__Organism|A|Concentration in container__' displayed as '__Concentration of A__'.
 
+<div class = "reportOutput">

```

Comment:
applying the style to the report output
@pchelle Theoretically it should also work like this:
`{r example 1 report, echo=FALSE, class.source='reportOutput'}`
But I didn't manage to get it to work then

Reply:
I think when I tried to look into that I also tried the Rmd code and it did not work on my side neither.
It is maybe not correctly translated into the html tag.
As long as the html tag does the job, I don't mind using it directly.